tensorflow/core/common_runtime/mkl_layout_pass_test.cc

```diff
@@ -4258,6 +4258,29 @@ TEST_F(MklLayoutPassTest, NodeRewrite_ConcatV2_DeviceTest) {
             "A->D:2;B->D;B:1->D:1;C->E;D->E:1");
 }
 
+
+TEST_F(MklLayoutPassTest, NodeRewrite_ConcatV2_idxtest) {
```

Comment:
Nit:
```suggestion
TEST_F(MklLayoutPassTest, NodeRewrite_ConcatV2_IndexTest) {
```

Reply:
@penpornk  Hi, thank you for the code review. I am covering the developer and will do code change today.  --GZ